UI/obs-app.cpp

```diff
@@ -1377,6 +1377,7 @@ bool OBSApp::OBSInit()
 	ProfileScope("OBSApp::OBSInit");
 
 	setAttribute(Qt::AA_UseHighDpiPixmaps);
+	setAttribute(Qt::AA_DontCreateNativeWidgetSiblings);
```

Comment:
This seems to interact badly with browser panels (screenshot from @WizardCM)
![](https://media.discordapp.net/attachments/374636084883095554/782841667975774227/unknown.png)

Reply:
Based on my quick testing, I'd like to avoid a global attribute like this one. Going through and changing it per-widget would be preferred.

Reply:
Images shows a tabbed dock with browser sources. Once you resize the window the tabs behave weirdly like shown. (for others like me who couldnt tell what was going on the image)

Reply:
Sadly this option does not exist as a non-global attribute AFAICT. The only thing I can think of is enabling WA_DontCreateNativeAncestors on widgets that have to be native and then enclosing them in some kind of no-op widget.

Reply:
This issue has been fixed via the change to `BrowserDock`.